build.xml

```diff
@@ -43,7 +43,8 @@
             patches/7491-preliminary.diff
             patches/7548_source-1.8.diff
             patches/7583_source-1.8.diff
-            patches/7621.diff 
+            patches/7621.diff
```

Comment:
See if you can avoid the minor whitespace change on this line so that merge conflicts are avoided. Maybe in a later commit we can remove the extra whitespace as it will not have any conflicts then.